ivy/func_wrapper.py

```diff
@@ -1365,6 +1365,7 @@ def _handle_nans(*args, **kwargs):
             elif nan_policy == "warns":
                 logging.warning("Nans are present in the input.")
 
+
```

Comment:
Let's revert this line here. lint-fixes shouldn't be part of this PR.

Reply:
Fixed